Assets/MixedRealityToolkit/_Core/Interfaces/IMixedRealityDevice.cs

```diff
@@ -1,35 +1,14 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-using Microsoft.MixedReality.Toolkit.Internal.Interfaces.InputSystem;
 
-namespace Microsoft.MixedReality.Toolkit.Internal.Definitions
+namespace Microsoft.MixedReality.Toolkit.Internal.Interfaces
 {
     /// <summary>
     /// Mixed Reality Toolkit device definition, used to instantiate and manage a specific device / SDK
     /// </summary>
-    public interface IMixedRealityDevice
+    public interface IMixedRealityDevice : IMixedRealityManager
     {
-        /// <summary>
-        /// Initialize function to setup the device, called after all managers are initialized by the Mixed Reality Manager
-        /// </summary>
-        void Initialize();
```

Comment:
Did we also remove these from all the scaffolding?

Reply:
It wasn't removed, it was simply replaced with the same methods in the Manager interface (which actually had more)